grisera_api/experiment/experiment_service_ontology.py

```diff
@@ -107,4 +107,24 @@ def delete_experiment(self, experiment_id: str):
         response = self.ontology_api_service.delete_instance(model_id, "Experiment", instance_label)
         if response["errors"] is not None:
             return ExperimentOut(experiment_name=instance_label, errors=response["errors"])
-        return ExperimentOut(experiment_name=response["label"])       
+        return ExperimentOut(experiment_name=response["label"])
+
+    def get_experiments(self):
+        """
+        Send request to ontology api to get experiments
+
+        Returns:
+            Result of request as list of experiments objects
+        """
+        model_id = 1
+        response = self.ontology_api_service.get_instances(model_id, "Experiment")
+        if response["errors"] is not None:
+            return ExperimentsOut(errors=response["errors"])
+
+        experiments = []
+        for inst in response["instances"]:
+            instance_name = inst["instance_name"]
+            instance = self.get_experiment(instance_name)
+            experiments.append(instance)
+
```

Comment:
Next time maybe you'd try list comprehension: 
```python
experiments = [self.get_experiment(inst["instance_name"]) for inst in response["instances"]]
```

Reply:
Changed and tested. it works well